src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -112,7 +108,7 @@ public WebCmdletElementCollection Links
                     EnsureHtmlParser();
 
                     List<PSObject> parsedLinks = new();
-                    MatchCollection linkMatch = s_linkRegex.Matches(Content);
+                    MatchCollection linkMatch = s_linkRegex!.Matches(Content!);
```

Comment:
"!" says we have bad implementation.  We could use a trick .Net uses - put these regex-s in helper static class so that it is initialized at first use time. I suggest implement this in new PR before.